internal/mode/static/state/graph/gateway_listener.go

```diff
@@ -238,8 +238,8 @@ func getAndValidateListenerSupportedKinds(listener v1.Listener) (
 
 	supportedKinds := make([]v1.RouteGroupKind, 0, len(listener.AllowedRoutes.Kinds))
 
-	validHTTPRouteKind := func(kind v1.RouteGroupKind) bool {
-		if kind.Kind != v1.Kind("HTTPRoute") {
+	validHTTPProtocolRouteKind := func(kind v1.RouteGroupKind) bool {
+		if kind.Kind != v1.Kind("HTTPRoute") && kind.Kind != v1.Kind("GRPCRoute") {
 			return false
 		}
 		if kind.Group == nil || *kind.Group != v1.GroupName {
```

Comment:
Does this also need to check v1alpha1 GroupName for now?

Reply:
It's the same `gateway.networking.k8s.io`